index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
This is a good start and you're handling the fs module properly.  I like that you've created an array of paths to read and are passing them into functions to get each file's data.  However, the way you've specified the readings doesn't guaruntee the files will be returned in the correct order, I suspect that since your files are similarly sized when you tested the code on your computer you recieved false positives.  Consider the following to understand the issue better:

var array = [];
array[2] = 'a';
console.log(array.length); // returns 3